server/service/async/async.go

```diff
@@ -69,7 +73,18 @@ func (t *Task) StartCollectors(ctx context.Context, jitterPct int, logger kitlog
 		errHandler:   collectorErrHandler,
 	}
 
-	colls := []*collector{labelColl, policyColl}
+	lastSeenColl := &collector{
+		name:         "collect_last_seen",
+		pool:         t.Pool,
+		ds:           t.Datastore,
+		execInterval: t.CollectorInterval,
```

Comment:
Does it make sense for host seen times to have same collector interval as the other two? (30s is the default IIRC)